src/PyroField.tsx

```diff
@@ -35,7 +35,7 @@ class PyroFieldInner<
     return this.props.children({
       core: {
         name: this.props.name,
-        value: this.props.values[this.props.name],
+        value: this.getValue(),
```

Comment:
I don't understand why we would put the error message in as the value of the field?